tests/unit/TestNetRemoteDataStreamingReactors.cxx

```diff
@@ -194,8 +205,10 @@ DataStreamReaderWriter::Await(uint32_t* numberOfDataBlocksReceived, Microsoft::N
         status.set_message("Timeout occurred while waiting for all operations to be completed");
         *m_readData.mutable_status() = std::move(status);
     }
+
     *numberOfDataBlocksReceived = m_numberOfDataBlocksReceived;
     *operationStatus = m_readData.status();
+    lostDataBlockSequenceNumbers = std::span<uint32_t>(std::data(m_lostDataBlockSequenceNumbers), std::size(m_lostDataBlockSequenceNumbers));
```

Comment:
This is modifying the value passed to the function, then not used. Was `lostDataBlockSequenceNumbers` meant to be a reference such that this update's the caller's arg?

Reply:
Yes, should be a reference. Will update